cg/services/orders/store_order_services/store_microbial_fastq_order_service.py

```diff
@@ -0,0 +1,132 @@
+from datetime import datetime
+
+from cg.models.orders.sample_base import StatusEnum
+
+from cg.constants import Workflow, DataDelivery
+from cg.models.orders.order import OrderIn
+
+from cg.services.orders.order_lims_service.order_lims_service import OrderLimsService
+from cg.services.orders.submitters.order_submitter import StoreOrderService
+from cg.store.exc import EntryNotFoundError
+from cg.store.models import Sample, Customer, Case, CaseSample
+from cg.store.store import Store
+
+
+class StoreMicrobialFastqOrderService(StoreOrderService):
+
+    def __init__(self, status_db: Store, lims_service: OrderLimsService):
+        self.status = status_db
+        self.lims = lims_service
+
+    def store_order(self, order: OrderIn) -> dict:
+        project_data, lims_map = self.lims.process_lims(lims_order=order, new_samples=order.samples)
+        status_data: dict = self.order_to_status(order)
+        self._fill_in_sample_ids(samples=status_data["samples"], lims_map=lims_map)
+        new_samples: list[Sample] = self.store_items_in_status(
+            customer_id=status_data["customer"],
+            order=status_data["order"],
+            ordered=project_data["date"] if project_data else datetime.now(),
+            ticket_id=order.ticket,
+            items=status_data["samples"],
+        )
+        return {"project": project_data, "records": new_samples}
+
+    @staticmethod
+    def order_to_status(order: OrderIn) -> dict:
+        """Convert order input for microbial samples."""
+        return {
+            "customer": order.customer,
+            "order": order.name,
+            "comment": order.comment,
+            "samples": [
+                {
+                    "application": sample.application,
+                    "comment": sample.comment,
+                    "data_analysis": sample.data_analysis,
+                    "data_delivery": sample.data_delivery,
+                    "name": sample.name,
+                    "priority": sample.priority,
+                    "volume": sample.volume,
```

Comment:
I don't think volume is something we store in StatusDB so it could probably be removed here?